### ISSUE TITLE
the iou always equals to 0.5 when I use my dataset to train the model

### ISSUE THREAD

**Issue by yangyang9706** [HasnainRaz/FC-DenseNet-TensorFlow#16]
Thank you for your work. I have the problem that the iou always equals to 0.5 when I use my dataset to train the model. I am looking forward to your answers.

**Comment by HasnainRaz** [comment]
Did you make any changes to the code?

**Comment by yangyang9706** [comment]
I didn't change the code

**Comment by HasnainRaz** [comment]
What format are your ground truth labels?

**Comment by yangyang9706** [comment]
the input image is 250*395, the mask is binarized images whose pixel values are 0 and 1.

**Comment by HasnainRaz** [comment]
You need to comment out the normalization function in the dataloader, since it divides the groundtruth labels by 255. Your labels are 0 and 1, 1/255 becomes 0, so the network never learns the foreground class, leading to a 0.5 iou.

**Comment by yangyang9706** [comment]
thank you so much! I have solved my problem!
